index.js

```diff
@@ -27,7 +39,70 @@ app.use(express.json());
 // IDEAS METHODS
 
 app.get('/ideas', (req, res) => {
-    res.status(200).json(getIdeas());
+    let sortedIdeasList;
+    if (req.query.sortBy === 'all') {
+        sortedIdeasList = getIdeas();
+    }
+    if (req.query.sortBy === 'totalDonationSum') {
+        sortedIdeasList = sortedByDonationSumIdeas();
+    }
+    if (req.query.sortBy === 'status' && req.query.status === 'accepted') {
+        sortedIdeasList = approvedIdeasList();
+    }
+    if (req.query.sortBy === 'status' && req.query.status === 'pending') {
+        sortedIdeasList = pendingIdeasList();
+    }
+    if (req.query.sortBy === 'status' && req.query.status === 'rejected') {
+        sortedIdeasList = rejectedIdeasList();
+    }
```

Comment:
Much better.

But now all 5 `if` will be executed - can you fix that?

Reply:
If I add return, it doesn't work.  That's strange.

Reply:
I think I've found the way out